config/initializers/sidekiq.rb

```diff
@@ -48,6 +48,8 @@ def self.connection_pool
   SidekiqUniqueJobs::Server.configure(config)
 end
 
+Sidekiq.logger.level = Rails.logger.level
```

Comment:
Currently, [Rails.logger.level](https://github.com/simpledotorg/simple-server/blob/724b90920c2b58d556ee5b6260455be02af51cb3/config/environments/production.rb#L55) is `info` level right? How will this set `error` log in Sidekiq?

Reply:
@PriyangaPKini the syntax you pointed to defaults to `info` in case env variable is missing. In production environment, it is set to `error`. @roypeter PR is good to for merging!

Reply:
Thanks for clarifying 👍